src/point_one/fusion_engine/messages/sta5635.h

```diff
@@ -83,6 +83,27 @@ struct P1_ALIGNAS(4) STA5635CommandResponse : public MessagePayload {
   uint8_t data[4] = {0};
 };
 
+/**
+ * @brief IQ data from an STA5635 sent in response to an @ref STA5635IQData.
```

Comment:
IQ data -> IQ sample data

STA5635 ~sent in response to an @ref STA5635IQData~ device.

This doxy block _is_ `STA5635IQData` so it can't be in response to it.